main.py

```diff
@@ -1,10 +1,45 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r", encoding="utf-8") as players_file:
+        players_info = json.load(players_file)
+
+    for player_name, player_data in players_info.items():
+
+        race_data = player_data.get("race")
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            description=race_data["description"]
+        )
+
+        skills_data = race_data.get("skills", [])
+        for skill_data in skills_data:
+            skill, _ = Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+
+        guild_data_ = player_data.get("guild")
```

Comment:
The variable name 'guild_data_' should be 'guild_data' to maintain consistency and avoid confusion. The underscore is unnecessary here.